### DISCUSSION TITLE
Graph?

### DISCUSSION THREAD

**Discussion by EmperorDalek** [taoteh1221/Slideshow_Crypto_Ticker#20] [Ideas]
Anyway to add some sort of 24 hour graph, maybe by default it could be disabled and enabled if desired, it could be some sort of transparent box (or background color changeable?) with a graph line through it for gain/lost, sort of like Robinhoods but much more simple.

The reason why I ask is because I am using the Longrunner 7" Touch LCD and there is quite a bit of room left on the screen (I don't really want to make the default ticker much larger because it becomes pixely and looks quirky that large) and a graph would add to functionality on larger displays (And smaller displays?).

https://www.amazon.com/gp/product/B071X8H5FB/ref=ppx_yo_dt_b_asin_title_o09_s00?ie=UTF8&psc=1

I know nothing of coding so take my idea with a grain of salt lol
Is this possible?
:)

**Comment by taoteh1221** [comment]
Sounds like your screen resolution is not set to match that display, if it looks pixely when you configure it to full screen size? 

An upcoming version will have an optional 4th row with Coingecko 1hr / 1d / 1w price change stats: https://github.com/taoteh1221/Slideshow_Crypto_Ticker/blob/main/DOCUMENTATION-ETC/TODO.txt#L84

Thanks for the feedback, just holler with any other questions and I'll help best I can. :)

**Reply by EmperorDalek** [reply]
Yeah, the pixelation is probably from the drivers, it's not a big deal to me.

Love the package, great job!
I do see your link there and that is great!

Would you have an ETA out of curiosity?
Also does this auto update or do we need to run update commands?

**Reply by taoteh1221** [reply]
Thanks for the compliments. Sorry, no ETA on this project's feature rollout. It's as I find time to implement it. You'll need to run the same command you ran to install it, and you're config is backed up in the same folder the config file is in (to migrate your custom settings into the new config).

**Reply by EmperorDalek** [reply]
Noted, thanks for the quick responses!

You answered my questions!
:)

**Reply by taoteh1221** [reply]
No prob, enjoy the ticker! :) I'll try to remember to post back here when I release that feature...guessing before summer / sometime this spring.
